openapi.yaml

```diff
@@ -19,6 +19,11 @@ externalDocs:
   url: https://github.com/osu-mist/financial-index-api
 produces:
   - application/json
+consumes:
+  - application/json
+security:
+  - OAuth2:
+      - full
```

Comment:
Looks like the openapi file is broken now. Can you fix it?

Reply:
I missed the securityDefinition. It is working now